Packs/CortexXDR/ReleaseNotes/4_10_43.md

```diff
@@ -0,0 +1,21 @@
+
+#### Classifiers
+
+##### Cortex XDR Incident Handler - Classifier
+
+- Updated the classifier to classify Port Scan alert as XDR Incident.
+- Fixed an issue with the classifier condition failing when `file_artifacts` is empty and not null.
+
+#### Playbooks
+
+##### Cortex XDR - Port Scan
+
+- Deprecated. Use the Cortex XDR - Port Scan - Adjusted playbook instead.
+
+##### Cortex XDR Alerts Handling
+
+- Fixed an issue with the condition due to mismatching letter case - Added ignore case.
```

Comment:
Mainly because I tried to follow the [templates](https://xsoar.pan.dev/docs/documentation/release-notes#general), but I changed it to fit both the conventions and your comment